pkg/accessor/events/events_accessor.go

```diff
@@ -0,0 +1,93 @@
+package events
+
+import (
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/eventbridge"
+	"github.com/aws/aws-sdk-go/service/eventbridge/eventbridgeiface"
+	"github.com/google/uuid"
+	"github.com/samber/lo"
+)
+
+type eventsAccessor struct {
+	client eventbridgeiface.EventBridgeAPI
+}
+
+// https://docs.aws.amazon.com/sdk-for-go/api/service/eventbridge/#EventBridge.PutRule
+func (this *eventsAccessor) CreateEventRule(name, eventBusName, eventPattern *string) (*string, error) {
+	putEventRuleInput := &eventbridge.PutRuleInput{
+		Name:         name,
+		EventBusName: eventBusName,
+		EventPattern: eventPattern,
+	}
+
+	response, err := this.client.PutRule(putEventRuleInput)
+
+	if err != nil {
+		return nil, err
+	}
+
+	ruleArn := response.RuleArn
+	return ruleArn, nil
+}
+
+func (this *eventsAccessor) PutEvents(eventBusName, detail, detailType, source, resourceName *string) (*eventbridge.PutEventsOutput, error) {
```

Comment:
go best practices is to use the name of the class as a single letter. e.g.
```suggestion
func (e *eventsAccessor) PutEvents(eventBusName, detail, detailType, source, resourceName *string) (*eventbridge.PutEventsOutput, error) {
```